CHANGELOG.rst

```diff
@@ -5,6 +5,51 @@ ibm.ibm_zos_core Release Notes
 .. contents:: Topics
 
 
+v1.6.0-beta.1
+=============
+
+Release Summary
+---------------
+
+Release Date: '2023-04-26'
+This changelog describes all changes made to the modules and plugins included
+in this collection. The release date is the date the changelog is created.
+For additional details such as required dependencies and availability review
+the collections `release notes <https://ibm.github.io/z_ansible_collections_doc/ibm_zos_core/docs/source/release_notes.html>`__
+
+Major Changes
+-------------
+
+- zos_volume_init - Introduces new module to handle volume (or minidisk) initialization. (https://github.com/ansible-collections/ibm_zos_core/pull/654)
+
+Minor Changes
+-------------
+
+- Updated the text converter import from "from ansible.module_utils._text" to "from ansible.module_utils.common.text.converters" to remove warning".. warn Use ansible.module_utils.common.text.converters instead.". (https://github.com/ansible-collections/ibm_zos_core/pull/602)
+- module_utils - job.py utility did not support positional wiled card placement, this enhancement uses `fnmatch` logic to support wild cards.
+- zos_copy - Fixed a bug where the module would change the mode for a directory when copying into it the contents of another. (https://github.com/ansible-collections/ibm_zos_core/pull/723)
```

Comment:
There is a bug fix inside the minor changes tag. I changed it for release v1.4.1 but apparently failed to communicate with the team, no need to run the pipeline again

Reply:
OK, maybe we can add it after this PR?